packages/mobile/src/redux/migrations.ts

```diff
@@ -387,6 +387,13 @@ export const migrations = {
     },
   }),
   30: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      sentryNetworkErrors: [],
```

Comment:
Not a big deal because biometry isn't live yet, but I think the order between 30 and 31 should be reversed :)
(ie biometry in 30 and network errors in 31)

Reply:
oops sorry i should have added a comment - this was a hack on purpose, i think we want to patch 1.25 with this PR and i accidentally merged biometry before this 🙈